### PR TITLE
Create About Us

### PR DESCRIPTION
**Pull request by The-Kergan**
# Who are we?

The team behind the toolkit are a group of volunteers from around the world. We have varied backgrounds, skills and experience, however we have one thing in common. We are determined to do what we can to provide everyone with practical tools to help protect their households and loved ones and reduce the spread of coronavirus.

# What are our Objectives

> To provide a free, pragmatic, evidence-based set of tools to combat COVID-19 for all, regardless of who you are or where you live in the world.

The tools will help anyone translate best health practice into clear actions any household or local community leader can take to reduce the risk of contracting COVID-19 or spreading it.

# Summary

The four main challenges to reducing coronavirus risk are massive in nature and global in scope. We must collectively use simple physical interventions to reduce the risks of the pandemic spread. The Coronavirus Community Toolkit aims to bridge the gap between ideal guidelines and actual behavior by making it easier for us all to keep our households safe.

# The problems we are facing

## Problem 1: Exponential pandemic growth

Worldwide, coronavirus deaths are growing exponentially, doubling every 6 days. (1) The authors break the data down by country because, _“some countries – like China and Korea – have very substantial countermeasures in place, and new daily confirmed deaths have declined. Many other countries do not have comparable measures in place, and numbers are quickly rising.”_

> Key Point: Nationwide spread reduction measures can dramatically slow the disease

## Problem 2: Vaccines are a long way off

Despite many scientists and organisations working flat out to produce a vaccine, scientists and policy makers believe that, _“we’ll be lucky to have a vaccine for use within 12-18 months.”_ (2)

> Key Point: Vaccines are not going to protect us in 2020

## Problem 3: Guidelines are hard to follow

Researchers have found that even Emergency Department policies during an influenza pandemic are not routinely followed, (3) and until recently, the response by political leaders and the general public to coronavirus social isolation guidelines has been slow or ignored entirely. (4, 5) 

> Key Point: We know guidelines are in place for a reason but they are often hard to find, hard to understand or hard to follow

## Problem 4: There are four barriers to crisis management

Concerned people want to do the right thing now to protect their households, however, there are four generic barriers to being able to manage the crisis effectively. (6) 

1. lack of imperative or precise wording
2. lack of easily identifiable instructions
3. lack of concrete performance targets, and 
4. lack of timely and adequate guidance on personal protective equipment and other safety measures

> Key Point: Not all of our leaders are behaving in a manner to lower barriers to crisis management

# What can we all do?

## Reserve health services for those most at risk

Approximately 80% of people who have COVID-19 do not have severe disease. 1 Acute health services are needed for the 20% with severe disease and the rest of their ‘normal’ acute patients. Most of the 80% of people who do contract coronavirus and do not have severe disease will recover at home. If your symptoms deteriorate, you should obviously seek acute care, so make sure you understand exactly what your government has told you to do in this circumstance.

## Use simple low cost interventions

Jefferson and colleagues concluded, ‘Simple and low-cost interventions would be useful for reducing transmission of epidemic respiratory viruses. Routine long-term implementation of some measures assessed might be difficult without the threat of an epidemic.’ (8) We are in a pandemic so this is our best chance to achieve high compliance with effective low cost interventions.

> Key Point: Until we have effective vaccines and drug treatments for coronavirus, our most effective measures to interrupt or reduce the spread of the virus are physical interventions like social-distancing and stay-at-home behaviors. (2, 3)

## Behave as if you and everyone you meet has coronavirus

No, of course not everyone has coronavirus, but it makes sense to behave as if they do. If you have any of the common symptoms of coronavirus behave as if you have coronavirus. If you don’t know what to do, follow the guidelines in your local country or from the World Health Organisation.

## Remove barriers to crisis management

It seems obvious, but we should also act to remove the barriers to effective crisis management identified by Timen and colleagues. (6) There has been a dramatic shift in recent weeks to communicate the imperative and to improve the availability of protective equipment, however, we are still hearing conflicting rhetoric from our leaders (imprecise wording) and the guidelines being published can be light on detail (try finding a consistent definition of what ‘emergency supplies’ are – is there any wonder we have seen extensive panic buying?).

# What is the team behind the Coronavirus Community Toolkit doing?

## Objective of the Toolkit

We are producing a free toolkit containing the best practical information we can find to try and lower Timen’s ‘barriers to crisis management’ for everyone. We believe that it will have most impact on the second item in the list (lack of easily identifiable instructions specific), and will focus on households and local community groups.

The tools in our toolkit will be supported by evidence because, as the WHO advised in February 2020, _"Rumours, misconceptions, and misinformation disseminated online via social media can have a negative impact on response measures and health-seeking behaviors."_ (10)

## Principles of the Toolkit

**Supported by Evidence:** We don’t want to add to the, _"rumours, misconceptions, and misinformation",_ and would prefer to use evidence based tools already available. We will focus most of our effort on developing specific ‘easily identifiable instructions’. We are calling each of these instructions a ‘Focus’ and each will be supported by multiple tools.

**Fast:** The CDC advocate that, ‘Scale and speed of response activities are paramount’, during an emergency (11) When anything is done fast quality tends to suffer. If you can prove that anything we produce is wrong or unclear, please let us know and we will update our tools – we actively encourage this!

# References

1. Roser M, Ritchie H, Ortiz-Ospina E, Coronavirus Disease (COVID-19) - Statistics and Research. https://ourworldindata.org/coronavirus#the-growth-rate-of-covid-19-deaths [cited 28/3/2020]
2. https://www.theguardian.com/world/2020/mar/27/inside-the-race-to-develop-a-coronavirus-vaccine-covid-19 [cited 28/3/2020]
3. Yu-Hsiang Hsieh et al. Emergency Physicians’ Adherence to Center for Disease Control and Prevention Guidance During the 2009 Influenza A H1N1 Pandemic. Western Journal of Emergency Medicine, Volume XIV, no. 2 : March 2013. DOI: 10.5811/westjem.2012.11.12246
4. https://www.theguardian.com/lifeandstyle/2020/mar/28/americans-who-dont-take-coronavirus-seriously [cited 28/3/2020]
5. https://www.bbc.com/news/uk-51882897
6. Timen A, Hulscher M, et al. Barriers to implementing infection prevention and control guidelines during crises: Experiences of health care professionals. American Journal of Infection Control, Volume 38, Issue 9, November 2010, Pages 726-733. doi.org/10.1016/j.ajic. 2010.03.006
7. Xianxian Zhao, Bili Zhang, Pan Li, Chaoqun Ma, Jiawei Gu, Pan Hou, Zhifu Guo, Hong Wu, Yuan Bai. Incidence, clinical characteristics and prognostic factor of patients with COVID-19: a systematic review and meta-analysis doi: https://doi.org/10.1101/2020.03.17.20037572
8. Jefferson T, Del Mar CB, Dooley L, Ferroni E, Al-Ansary LA, Bawazeer GA, van Driel ML, Nair S, Jones MA, Thorning S, Conly JM. Physical interventions to interrupt or reduce the spread of respiratory viruses. Cochrane Database of Systematic Reviews 2011, Issue 7. Art. No.: CD006207. DOI: 10.1002/14651858.CD006207. pub4.
9. Fong MW, Gao H, Wong JY, Xiao J, Shiu EYC, Ryu S, et al. Nonpharmaceutical measures for pandemic influenza in nonhealthcare settings—social distancing measures. Emerg Infect Dis. 2020 May [cited 25/3/2020].
10. https://www.who.int/docs/default-source/coronaviruse/srp-04022020.pdf
11. https://www.cdc.gov/eis/field-epi-manual/chapters/EOC-Incident-Management.html

### CODE REVIEW

**Comment by The-Kergan** [comment]
Hi - I created this as a draft. How do I place this in the header of the site behind a link called 'About Us'?
@geoffwasham

**Comment by zeue** [comment]
@The-Kergan I will do that for you now.

File: docs/about.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+> Key Point: Vaccines are not going to protect us in 2020
+
+## Problem 3: Guidelines are hard to follow
+
+Researchers have found that even Emergency Department policies during an influenza pandemic are not routinely followed, (3) and until recently, the response by political leaders and the general public to coronavirus social isolation guidelines has been slow or ignored entirely. (4, 5) 
+
+> Key Point: We know guidelines are in place for a reason but they are often hard to find, hard to understand or hard to follow
+
+Problem 4: There are four barriers to crisis management
```
Comment (The-Kergan) [inline docs/about.md:35]:
```suggestion
## Problem 4: There are four barriers to crisis management
```

File: docs/about.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+The tools in our toolkit will be supported by evidence because, as the WHO advised in February 2020, ‘Rumours, misconceptions, and misinformation disseminated online via social media can have a negative impact on response measures and health-seeking behaviors.’ (10)
+
+## Principles of the Toolkit
+
+*Supported by Evidence:* We don’t want to add to the, ‘rumours, misconceptions, and misinformation’, and would prefer to use evidence based tools already available. We will focus most of our effort on developing specific ‘easily identifiable instructions’. We are calling each of these instructions a ‘Focus’ and each will be supported by multiple tools.
+
+*Fast:* The CDC advocate that, ‘Scale and speed of response activities are paramount’, during an emergency (2) When anything is done fast quality tends to suffer. If you can prove that anything we produce is wrong or unclear, please let us know and we will update our tools – we actively encourage this!
```
Comment (The-Kergan) [inline docs/about.md:78]:
```suggestion
*Fast:* The CDC advocate that, ‘Scale and speed of response activities are paramount’, during an emergency (11) When anything is done fast quality tends to suffer. If you can prove that anything we produce is wrong or unclear, please let us know and we will update our tools – we actively encourage this!
```

File: docs/about.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+> Key Point: Until we have effective vaccines and drug treatments for coronavirus, our most effective measures to interrupt or reduce the spread of the virus are physical interventions like social-distancing and stay-at-home behaviors. (2, 3)
+
+## Behave as if you and everyone you meet has coronavirus
+
+No, of course not everyone has coronavirus, but it makes sense to behave as if they do. If you have any of the common symptoms of coronavirus behave as if you have coronavirus. If you don’t know what to do, follow the guidelines in your local country or from the World Health Organisation.
+
+## Remove barriers to crisis management
+
+> It seems obvious, but we should also act to remove the barriers to effective crisis management identified by Timen and colleagues. (6) There has been a dramatic shift in recent weeks to communicate the imperative and to improve the availability of protective equipment, however, we are still hearing conflicting rhetoric from our leaders (imprecise wording) and the guidelines being published can be light on detail (try finding a consistent definition of what ‘emergency supplies’ are – is there any wonder we have seen extensive panic buying?).
```
Comment (The-Kergan) [inline docs/about.md:64]:
```suggestion
It seems obvious, but we should also act to remove the barriers to effective crisis management identified by Timen and colleagues. (6) There has been a dramatic shift in recent weeks to communicate the imperative and to improve the availability of protective equipment, however, we are still hearing conflicting rhetoric from our leaders (imprecise wording) and the guidelines being published can be light on detail (try finding a consistent definition of what ‘emergency supplies’ are – is there any wonder we have seen extensive panic buying?).
```
